src/css/style.css

```diff
@@ -2403,3 +2405,9 @@ a:only-of-type > .remove {
 :root.gallery-open.fixed #header-bar:not(.autohide) #shortcuts .fa::before {
   visibility: hidden;
 }
+.capcodeJumper > .prev,
+.capcodeJumper > .next,
+.uniqueIDJumper > .prev,
+.uniqueIDJumper > .next {
+  font-size: 120%;
+}
```

Comment:
Would probably be good to have a generic `jumper` class to avoid the need for duplicate CSS like this.
Also needs a comment to indicate the block of CSS related to PostJumper, similar to what's done in this file for other features.